ui/lib/admin.js

```diff
@@ -1,11 +1,15 @@
 import useSWR from 'swr'
 
+import { useGrants } from './grants'
+
 export function useAdmin () {
```

Comment:
Does this get called every time a page is loaded? Wondering if it should just be cached somewhere.

Reply:
It gets cached by [`swr`](https://swr.vercel.app/)

Reply:
This is a great catch! It was still revalidating on every page. Fixed